odc/stats/plugins/lc_level34.py

```diff
@@ -94,7 +85,7 @@ def reduce(self, xx: xr.Dataset) -> xr.Dataset:
             xx, self.bare_threshold, veg_cover
         )
 
-        l4 = l4_natural_aquatic.natural_auquatic_veg(l4, veg_cover, water_seasonality)
+        l4 = l4_natural_aquatic.natural_auquatic_veg(l4, veg_cover, xx.water_season)
```

Comment:
water seasonality comes from `frequency` band of wo summaries, thresholded by 0.25, instead of `water_season` band of level 1

Reply:
As mentioned above